### DISCUSSION TITLE
PID killed plots

### DISCUSSION THREAD

**Discussion by elmans1961** [Chia-Network/chia-blockchain#4939] [Q&A]
Are PID killed plots  damaged ones,useless and should be deleted?

**Comment by ishtob** [comment]
if the plotting is not complete, then yes. if you killed via PID after phase 4 is complete and it is still moving the temp plot over to the final directory, then you may be able t save it by renaming **.plot.2.tmp to **.plot

i would run a chia plots check -g ***(name of plot) to check if it is functional after

**Reply by elmans1961** [reply]
Thank you!I checked my plots after I foun in log file that some plots are  PID killed plots.According to my checking(chia plots check -g) all plots are elligible.Write,please,how should I rename a plot if smth happens with it?
